tests/Http/HttpRequestTest.php

```diff
@@ -537,4 +537,22 @@ public function testCreateFromBase()
 
         $this->assertEquals($request->request->all(), $body);
     }
+
+    public function testJsonContent()
+    {
+        $body = [
+            'foo' => 'bar',
+            'baz' => ['qux'],
+        ];
+
+        $server = [
+            'CONTENT_TYPE' => 'application/json',
```

Comment:
Ah right. Bit strange of them to duplicate that logic. Maybe we should send a PR to remove that?